figures/mapping-histology-labels.Rmd

```diff
@@ -50,17 +52,25 @@ library(tidyverse)
 library(RColorBrewer)
 ```
 
+
+We should perform some minimal checking to make sure the provided release is _at least_ at version 21.
+```{r}
+minimum_allowed_release <- 21
+release_version <- as.numeric(stringr::str_match(params$release, "-v(\\d+)-")[[2]])
+stopifnot(release_version >= minimum_allowed_release)
```

Comment:
Is this going to give us an informative message?

Reply:
No, good point. Just changed to `if` and `stop()` to indeed print an informative error.